tests/mocks/utils.mock.ts

```diff
@@ -14,6 +15,7 @@ export function initBashSpy (spyMocks: {cmd: string; returnValue: any}[]) {
 
     for (const spyMock of spyMocks) {
         when(spyOn).calledWith(spyMock.cmd, expect.any(String)).mockResolvedValue(spyMock.returnValue);
+        when(spyOn).calledWith(spyMock.cmd).mockResolvedValue(spyMock.returnValue);
```

Comment:
So that mocking works when the functions are called without the `cwd` args 
ie. 
`Utils.bash("ls")`